group_vars/openstack/gluster

```diff
@@ -17,7 +17,7 @@ gluster_group_servers:
   flavor: "j1.small"
   image: "centos-7-20190104"
   user: "centos"
-  num_nodes: 3
+  num_nodes: "{{ gluster_num_nodes }}"
```

Comment:
Should we define a value for `gluster_num_nodes`, or for consistency should we remove the value set for `beegfs_oss_num_nodes` in `group_vars/openstack/beegfs`

Reply:
@oneswig Have we tested resizing a gluster cluster? If not, then we should fix it at three.

Or we could just default it to that in the ui-meta. Does gluster have the requirement that the number of nodes is odd and > 3, like a lot of HA stuff (e.g. etcd).

I don't think I have a problem with fixed-size storage clusters.

Reply:
Hmm resizing appears to fail with cluster unfortunately...

Reply:
I have set the min and max nodes to 3 until we manage to fix the scaling which I can look into now.

Reply:
BeeGFS seems to scale okay.